rkt/api_service.go

```diff
@@ -82,6 +82,14 @@ func (s *v1AlphaAPIServer) GetInfo(context.Context, *v1alpha.GetInfoRequest) (*v
 			RktVersion:  version.Version,
 			AppcVersion: schema.AppContainerVersion.String(),
 			ApiVersion:  supportedAPIVersion,
+			GlobalFlags: &v1alpha.GlobalFlags{
+				Dir:                globalFlags.Dir,
```

Comment:
@iaguis 
> If I understand this correctly, the intention of this PR is expose the options used to call the rkt API service through the API service itself. Then, something (the kubelet) can query the API service and use those options when spawning new pods.

Right.

I think `getDataDir()` is ok, because we want to pass the data dir which the rkt api service is using, so that when creating a pod, it can be detected by rkt api service.